dotnet/src/Connectors/Connectors.Memory.Weaviate/Http/ApiSchema/ObjectResponseResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Nodes;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Weaviate.Http.ApiSchema;
+
+// ReSharper disable once ClassNeverInstantiated.Global
+internal sealed class ObjectResponseResult
+{
+    public JsonObject? Errors { get; set; } // TODO? should this be an object?
```

Comment:
Should this TODO remain?

Reply:
Its OK to remain as a JsonObect. I will remove the TODO